client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -1,19 +1,20 @@
 # -*- coding: utf-8 -*-
 
-import json
 import gzip
-import os
-import time
-from urllib.parse import urlparse
+import json
 import warnings
 
-import requests
+from requests import Session
+from typing import Any, Dict, List, Optional, Set, Tuple
+from urllib3 import Retry
+from urllib.parse import urlparse
+from verta import credentials
```

Comment:
See other comments about import order or `typing`, `urllib`, and `verta` (which maybe could be changed to `..`).

Reply:
Import order fixed in all places.  Apologies for the confusion